server/models/Profile.js

```diff
@@ -0,0 +1,41 @@
+const mongoose = require("mongoose");
+
+const reqString = {
+    type: String,
+    require: true
+};
+
+const ProfileSchema = new mongoose.Schema({
+    userId: {
+        type: mongoose.Schema.Types.ObjectId,
+        require: true,
+        ref: 'user'
+    },
+    firstName: reqString,
+    lastName: reqString,
+    email: reqString,
```

Comment:
The email is already in the User model. You can populate the `userId` instead when you need the email to avoid conflicts.